android/build.gradle

```diff
@@ -42,9 +42,7 @@ android {
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 
-    implementation 'com.iterable:iterableapi:3.4.2'
-    implementation 'com.google.firebase:firebase-messaging:22.0.0'
```

Comment:
Is this implementation not required anymore?

Reply:
I tested and it works fine without that, why was that required?